playwright/pom.xml

```diff
@@ -73,5 +73,9 @@
       <groupId>com.microsoft.playwright</groupId>
       <artifactId>driver-bundle</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-junit-jupiter</artifactId>
```

Comment:
I hope adding Mockito as a test dependency is okay.  For soft assertion test, all we need to verify is that we are calling the correct underlying assertion method instead.  So I'm using Mockito for that.